scripts/inject-dict.json

```diff
@@ -124,5 +124,11 @@
   {
     "tpl": "polkadot_max_nominations",
     "default": "16"
+  },
+  {
+    "tpl": "vote_locking_period",
+    "default": { "kusama": 115200, "polkadot": 403200 },
+    "path": "consts.democracy.voteLockingPeriod",
+    "filters": ["blocksToDays"]
```

Comment:
Ah okay this is good to know.  I also see from your previous commit that the table is still only rendering Polkadot values but using conditional rendering with a table doesn't seem to work.  I was trying to see if I could find another existing example in the repo of this.